### PR TITLE
Allow uploads of dll files

### PR DESCRIPTION
**Pull request by chidozieononiwu**
- Allow upload of dll files for C#
UploadInstructions

### CODE REVIEW

**Review by praveenkuttappan — CHANGES_REQUESTED** [review]
As discussed, let's fix the label to correctly convey the message.

**Comment by chidozieononiwu** [comment]
> As discussed, let's fix the label to correctly convey the message.

I have updated it.

File: src/dotnet/APIView/ClientSPA/src/app/_components/revisions-list/revisions-list.component.ts [hunk 1]
```diff
@@ -506,7 +507,8 @@ export class RevisionsListComponent implements OnInit, OnChanges {
           `Run <code>dotnet pack</code>`, 
           `Upload the resulting .nupkg file.`
         ];
-        this.acceptedFilesForReviewUpload = ".nupkg";
+        this.acceptedFilesForReviewUpload = ".nupkg, .dll";
+        this.acceptedFilesLabelForReviewUpload = ".nupkg or .dll";
```
Comment (maririos) [inline src/dotnet/APIView/ClientSPA/src/app/_components/revisions-list/revisions-list.component.ts:511]: could you add a screen that shows how this is displayed in the overall message?

**Comment by maririos** [comment]
Thanks! looking at the image, I have 2 suggestions:
- Update the # 2 instructions to "Upload the resulting .nupkg or dll file"
- Update the message on the button to "Upload file" or "Select file"

**Review by praveenkuttappan — APPROVED** [review]

**Pull request merged**